pkgs/applications/networking/irc/convos/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchFromGitHub, perl, perlPackages, shortenPerlShebang, makeWrapper }:
+
+with stdenv.lib;
+
+perlPackages.buildPerlPackage rec {
+  pname = "convos";
+  version = "4.18";
```

Comment:
Just out of curiosity: Is there a way to make this dynamic? Sorry if this is off topic and something I should look up or ask in a different channel.

Reply:
Thx, replaced with the text on the website.